test/cypress/executor-smoke/crd/crd.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     repository:
       type: git
       uri: https://github.com/kubeshop/testkube
-      branch: main
+      branch: executor-tests-run
```

Comment:
Why do we switch to custom branch?

Reply:
I used it temporary for testing video param - it's already changed in the last commit